src/BenchmarksServer/Startup.cs

```diff
@@ -2348,6 +2348,9 @@ private static async Task<string> CloneRestoreAndBuild(string path, ServerJob jo
                 throw new NotSupportedException($"Unsupported framework: {targetFramework}");
             }
 
+            // #1445 force no cache for restore to avoid restore failures for packages published within last 30 minutes
+            buildParameters += "/p:RestoreNoCache=true ";
```

Comment:
Does it mean that a miss is actually cached?

Reply:
According to https://github.com/dotnet/sdk/issues/9577#issuecomment-403985521

> nuget restore has default HTTP cache of 30min

So if you try to restore a NuGet package that has been deployed to NuGet feed in the last 30 minutes, the restore is going to... fail. With this change, it's forced not to cache.